taxonomies/ingredients.txt

```diff
@@ -29826,7 +29846,7 @@ cy:Soia
 da:soja
 de:Soja
 el:σόγια
-es:soja, soya
+es:soja, soya, soja-ecologica
```

Comment:
We should not add "ecologica" to entries. Instead we will improve the parser so that it recognizes the label for all ingredients.

Reply:
ok